app/routes/event.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const auth = require('../middleware/auth')
+const router = express.Router()
+const eventController = require('../controllers/event')
+
+// CREATE A POST
+router.post(
+  '/',
+  auth,
+  eventController.createEvent
+)
+
+router.post(
+  '/event_update/:id',
+  auth,
+  eventController.updateEvent
+)
+router.post(
+  '/:id/event_rsvp/:event_id',
+  auth,
+  eventController.rsvpOfEvent
+)
+router.get(
+  '/geteventbyId/:id',
```

Comment:
`/geteventbyId/:id`
no need of `geteventbyId/` it should be `/:id`
Please read about `REST`